### ISSUE TITLE
Docker run failing on parse.worker.js

### ISSUE THREAD

**Issue by marcmtk** [nextstrain/nextclade#233]
Hi all,

First off, thank you for your work on this package 👍 

I've been trying to run nextclade on the command line, preferably via docker.

```
>docker run -it --rm --volume="C:\Users\Marc\Downloads:/seq" neherlab/nextclade nextclade.js --input-fasta /seq/samlet.fasta --output-json /seq/samlet.json
Error: Cannot find module '/usr/local/bin/parse.worker.js'
Require stack:
- /usr/local/bin/nextclade.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:892:15)
    at Function.resolve (internal/modules/cjs/helpers.js:94:19)
    at resolveScriptPath (/usr/local/bin/nextclade.js:1059:2646)
    at new Worker (/usr/local/bin/nextclade.js:1059:3106)
    at _callee$ (/usr/local/bin/nextclade.js:904:2152)
    at tryCatch (/usr/local/bin/nextclade.js:419:1007)
    at Generator.invoke [as _invoke] (/usr/local/bin/nextclade.js:419:4958)
    at Generator.next (/usr/local/bin/nextclade.js:419:2146)
    at asyncGeneratorStep (/usr/local/bin/nextclade.js:424:275)
    at _next (/usr/local/bin/nextclade.js:424:586) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/usr/local/bin/nextclade.js' ]
}
```
Going to the location in the container didn't yield any hints for me.
```
>docker run -it --rm neherlab/nextclade bash
root@4b8ba35d1d90:/# cd /usr/local/bin
root@4b8ba35d1d90:/usr/local/bin# ls -la
total 73564
drwxr-xr-x 1 root staff     4096 Oct 13 06:51 .
drwxrwsr-x 1 root staff     4096 Sep  1 16:02 ..
-rwxrwxr-x 1 root root       116 Sep  1 16:02 docker-entrypoint.sh
lrwxrwxrwx 1 root root        12 Oct 13 06:51 nextclade -> nextclade.js
-rwxrwxr-x 1 root root   1612452 Oct 13 06:50 nextclade.js
-rwxr-xr-x 1 root staff 73700808 Aug 27 13:04 node
lrwxrwxrwx 1 root root        19 Sep  1 16:02 nodejs -> /usr/local/bin/node
lrwxrwxrwx 1 root staff       38 Aug 27 13:04 npm -> ../lib/node_modules/npm/bin/npm-cli.js
lrwxrwxrwx 1 root staff       38 Aug 27 13:04 npx -> ../lib/node_modules/npm/bin/npx-cli.js
lrwxrwxrwx 1 root root        26 Sep  1 16:02 yarn -> /opt/yarn-v1.22.5/bin/yarn
lrwxrwxrwx 1 root root        29 Sep  1 16:02 yarnpkg -> /opt/yarn-v1.22.5/bin/yarnpkg
```

Can you offer any help with this matter?

Kind regards
Marc

**Comment by ivan-aksamentov** [comment]
@marcmtk Hi Marc, thanks for the report!

Looks like we forgot to publish some of the new files in the latest release.

Could you please tell me what version does it report?
```
docker run -it --rm neherlab/nextclade nextclade --version
```

Meanwhile feel free to roll back to version 0.7.5:

```
docker run -it --rm neherlab/nextclade:0.7.5 nextclade --version

```

This version does not need the new files.

**Comment by marcmtk** [comment]
```
> docker run -it --rm neherlab/nextclade nextclade --version
0.8.0-alpha.1
```
Thank you. Rolling back to 0.7.5 did indeed solve my problem 🎉 😄

**Comment by ivan-aksamentov** [comment]
This should be fixed now.

The `latest` tag on docker hub will no longer refer to the last published alpha and beta versions, only to releases.

I have:
 - republished the `latest` tag (and related tags like `stretch`, `alpine` etc.) with the last stable release `0.7.5`
 - made sure that alphas and betas are pushed to their corresponding tags (in 
https://github.com/nextstrain/nextclade/commit/7ea81b19d1cfdeaccd10533decb3ae4b5697ea30)
 - made sure that all necessary files are included into the docker builds (in https://github.com/nextstrain/nextclade/commit/dbb54f2a8584379437932d7480cc08db34d16a87)
 - released `0.8.0-alpha.2`, which includes previously missing files

P.S. @marcmtk these missing files are WebWorker modules - we've recently made  nextclade CLI to run sequence analysis in parallel. If you have large workloads, this can speedup things significantly. We'd appreciate your early feedback. If you have a chance, you can try the new alpha:

```
docker run -it --rm neherlab/nextclade:0.8.0-alpha.2 nextclade --jobs=8 ...
```

**Comment by marcmtk** [comment]
I just tried running 0.8.0-alpha.2 on my local windows machine and a remote HPC through singularity. Both worked 👍  
On windows i can tell that it loaded all the cores for a moment.

I tried timing the runs on both systems, but the results were quite variable, i don't think i have enough sequences (~30 currently) to really test the timings.
